lib/cog/commands/group/create.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Cog.Commands.Group.Create do
+  require Cog.Commands.Helpers, as: Helpers
+  alias Cog.Repository.Groups
+
+  Helpers.usage """
+  Creates new user groups.
+
+  USAGE
+    group create [FLAGS] <group_name>
+
+  ARGS
+    group_name    The name of the user group to create
+
+  FLAGS
+    -h, --help    Display this usage info
+  """
+
+  @spec create_group(%Cog.Command.Request{}, List.t) :: {:ok, String.t, Map.t} | {:error, any()}
+  def create_group(req, arg_list) do
+    if Helpers.flag?(req.options, "help") do
```

Comment:
The helper is nice, but it might actually be a bit nicer to use pattern matching for this stuff. `def create_group(%{options: %{"help" => true}}, _args)` is clear enough to me at least.